src/RenderWebGL.js

```diff
@@ -717,9 +732,18 @@ class RenderWebGL extends EventEmitter {
 
         const bounds = this._candidatesBounds(candidates);
 
-        // if there are just too many pixels to CPU render efficently, we
-        // need to let readPixels happen
-        if (bounds.width * bounds.height * (candidates.length + 1) >= __cpuTouchingColorPixelCount) {
+        const maxPixelsForCPU = (typeof this._forceGPU === 'boolean') ?
```

Comment:
Having these two `_forceGPU` tests feels weird to me. Would testing explicitly `=== true`, `=== false`, or `=== null` be better?